Dartmouth Coach/TicketsViewController.swift

```diff
@@ -11,13 +11,125 @@ import UIKit
 class TicketsViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, TicketTableViewCellDelegate {
 
     @IBOutlet weak var tableView: UITableView!
+    var prefs: NSUserDefaults!
+    var tickets = [NSData]()
+    let demo = true
+    
+    /*
+        Data structure for tickets
+    */
+    final class TicketData: NSObject, NSCoding {
```

Comment:
If you put this class inside the other class then you can't use the object outside of it? I'm just assuming not sure how it works...